aleph-client/src/contract/convertible_value.rs

```diff
@@ -193,3 +186,124 @@ where
         }
     }
 }
+
+impl<Elem: TryFrom<ConvertibleValue, Error = anyhow::Error>> TryFrom<ConvertibleValue>
+    for Vec<Elem>
+{
+    type Error = anyhow::Error;
+
+    fn try_from(value: ConvertibleValue) -> std::result::Result<Self, Self::Error> {
```

Comment:
Question:

```suggestion
    fn try_from(value: ConvertibleValue) -> Result<Self> {
```

should work here, no?

Reply:
right - I simplified it in every place in the module